general-concepts/licenses/text.xml

```diff
@@ -19,6 +39,85 @@ be used) then use the following syntax:
 LICENSE="|| ( foo bar )"
 </codesample>
 
+<subsection>
+<title>Determining the correct license</title>
+<body>
+
+<warning>
+Do not trust the license indicated by GitHub or other sites that use
+automation to guess the license! The information presented is rarely
+complete, and may be incorrect.
+</warning>
+
+<p>
+To establish the correct value of <c>LICENSE</c>, you need to trace
+the licenses of all installed files. Normally, the licenses of output
+files (compiled executables, generated files) are implied
+by the licenses of the respective input files.
+</p>
+
+<p>
+When looking for license information, the following should be
+considered:
+</p>
+
+<ol>
+	<li>
+		<c>COPYING*</c> and <c>LICENSE*</c> files distributed with
+		the package
+	</li>
+	<li>
+		explicit statements in documentation
+	</li>
+	<li>
+		explicit license notices in source and data files
+	</li>
+</ol>
+
+<p>
+The latter (more specific) options take precedence over the former.
+In particular, <c>COPYING*</c> files are frequently included as hardcopy
+of applicable licenses but the exact application of licenses and their
+versions is specified elsewhere.
+</p>
+
+<p>
+Please watch for license conflicts. If the license indicated
+by the package is incompatible with the licenses used by its sources
+(e.g. BSD/MIT package including GPL sources), please contact
+the licenses team for guidance. Do not add packages that seem
+to include license term violations.
+</p>
+
+</body>
+</subsection>
+
+<subsection>
+<title>GPL-x vs GPL-x+</title>
+<body>
+
+<p>
+FSF licenses (GPL, LGPL, AGPL, FDL) occur in two variants: the 'vN only'
+and 'vN or later' variants. In Gentoo, the latter are followed by a plus
+sign (e.g. <c>GPL-2+</c>), while the former are not (e.g. <c>GPL-2</c>).
+</p>
+
+<p>
+Determining the correct variant usually requires looking for copyright
+notices in the code. For example, the following copyright notice
+indicates <c>GPL-2+</c> license:
+</p>
+
+<pre>
+ * This program is free software; you can redistribute it and/or 
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of 
+ * the License, or (at your option) any later version.
+</pre>
+
```

Comment:
Should we continue and contrast this by following up with the GPL-2 only clause?

Reply:
I don't think that's strictly necessary. Nor that there is an official 'GPL-2 only' clause version, people make it as they go.